### PR TITLE
Fix Combobox variant

### PR DESCRIPTION
**Pull request by vorillaz**
### Background
Added "solid" variant for Combobox and MultiCombobox, also added a small fix for DateRangeInputProps as TS is complaining.

### Testing

- `npm run test`

### CODE REVIEW

File: src/components/Combobox/Combobox.tsx [hunk 1]
```diff
@@ -80,6 +83,7 @@ function Combobox<Item>({
   onChange,
   value,
   items,
+  variant = 'outline',
```
Comment (alexmylonas) [inline src/components/Combobox/Combobox.tsx:86]: This design is the one we already display right? i am asking so dont have to go update everything!

Comment (alexmylonas) [reply src/components/Combobox/Combobox.tsx:86]: From what i am seeing that's a case, just trying to confim it.

File: src/components/DateRangeInput/DoubleTextInput.tsx [hunk 2]
```diff
@@ -24,6 +24,7 @@ export type DoubleTextInputProps = Omit<TextInputProps, 'label' | 'placeholder'>
   placeholderTo?: string;
 
   /** The `from` value for the double text input*/
+
```
Comment (alexmylonas) [inline src/components/DateRangeInput/DoubleTextInput.tsx:27]: nitpick: No need for this extra line

File: src/components/MultiCombobox/MultiCombobox.tsx [hunk 3]
```diff
@@ -262,7 +277,7 @@ function MultiCombobox<Item>({
               <InputControl
                 invalid={invalid}
                 disabled={disabled}
-                variant={isOpen && items.length ? 'solid' : 'outline'}
+                variant={multiComboboxVariant}
```
Comment (alexmylonas) [inline src/components/MultiCombobox/MultiCombobox.tsx:280]: Should we still account for the `items.length`?

**Review by alexmylonas — APPROVED** [review]
Solid work! (pun intended) 😛 

I am approving but please have a quick look at my comments!

File: src/components/Combobox/Combobox.tsx [hunk 4]
```diff
@@ -101,6 +105,15 @@ function Combobox<Item>({
   // plus some focus/blur events that are tied to the searchable behaviour (see below)
   const [inputValue, setInputValue] = React.useState('');
 
+  const getVariant = React.useCallback(
+    isOpen => {
```
Comment (3nvi) [inline src/components/Combobox/Combobox.tsx:109]:
```suggestion
    (isOpen: boolean) => {
```

File: src/components/MultiCombobox/MultiCombobox.tsx [hunk 5]
```diff
@@ -130,6 +134,15 @@ function MultiCombobox<Item>({
   hidden,
   ...rest
 }: MultiComboboxProps<Item>): React.ReactElement<MultiComboboxProps<Item>> {
+  const getVariant = React.useCallback(
+    isOpen => {
```
Comment (3nvi) [inline src/components/MultiCombobox/MultiCombobox.tsx:138]:
```suggestion
    (isOpen: boolean) => {
```

**Review by 3nvi — APPROVED** [review]
LGTM

File: src/components/Combobox/Combobox.tsx (hunk 1 above)
Comment (vorillaz) [reply src/components/Combobox/Combobox.tsx:86]: Yeap, that's intended! Thanks for the comment.

File: src/components/MultiCombobox/MultiCombobox.tsx (hunk 3 above)
Comment (vorillaz) [reply src/components/MultiCombobox/MultiCombobox.tsx:280]: Nice catch!

**Pull request merged**